<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>673</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnAnalyzeScene">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Analyze Current Scene</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbbFileFormat"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbbFilter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeViewResult">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSelectMissingTextures">
     <property name="text">
      <string>Select  Missing Textures</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnAssigntoDirectories">
     <property name="text">
      <string>Assign to another Directories</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QPushButton" name="btnchangeFormat">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Change Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbbTargetType">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Find:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ldtOldName">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 4px;
border-style: solid;
border: 1px solid gray;
border-radius: 10px;
font:  bold 11px
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ldtNewName">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 4px;
border-style: solid;
border: 1px solid gray;
border-radius: 10px;
font:  bold 11px
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSelect_Missing_Files">
   <property name="text">
    <string>Select missing files in folder</string>
   </property>
  </action>
  <action name="actionAssign_to_Another_path">
   <property name="text">
    <string>Assign to another path</string>
   </property>
  </action>
  <action name="actionChange_Format">
   <property name="text">
    <string>Change format</string>
   </property>
  </action>
  <action name="actionRename_File">
   <property name="text">
    <string>Rename file</string>
   </property>
  </action>
  <action name="actionSelect_Textures_Inside">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
